### PR TITLE
fixed issue #9

### PR DESCRIPTION
**Pull request by pixelverbieger**
- added text domain where the ticket mentioned (3 lines)
- added text domain in plugin header (serverstate.php)

### CODE REVIEW

File: inc/serverstate.class.php [hunk 1]
```diff
@@ -167,7 +167,7 @@ public static function add_action_link($input)
 						),
 						admin_url('/')
 					),
-					__('Settings')
+					__( 'Settings', 'serverstate' )
```
Comment (bueltge) [inline inc/serverstate.class.php:170]: Small hint, we should sanitize strings from translation files, so we should use `esc_html__` instead of `__`.

**Comment by bueltge** [comment]
Thanks for your request!
However I can't find an method, function to load translation files? So that your changes should not work, load a translation for the custom textdomain. Maybe you should enhance this?

**Comment by pixelverbieger** [comment]
Hallo Frank, da ich von OOP keine Ahnung habe, lasse ich da die Finger von :)

**Comment by stklcode** [comment]
That's no magic. Just add a call to `load_plugin_textdomain( 'serverstate' );`
somewhere in the `__construct()` Method of the `Serverstate` class, after the shotrhand returns for autosave, ajax, etc. to ensure it is loaded before any calls to plugin.

If it's only required for admin pages, you could consider moving it into the `if (is_admin()) { ... }` block to reduce overhead, otherwise before this. (haven't checked whether this might be suitable here).

**Comment by krafit** [comment]
Thank you everyone for your time and effort on this. Due to #12 we won't merge any further pull requests

**Pull request closed without merging**